20220708.md

```diff
@@ -43,22 +43,23 @@ A lot of repositories (several tens) has seen the creation of a new `upgrade-4.4
 
 ## Browser stack
 
-* `sailfish-browser`, the browser application itself, **rainemak** commented on **dcaliste** pull request to [introduce a long-tap in tab list view](https://github.com/sailfishos/sailfish-browser/pull/959) for house-keeping like tab deletion. **mikaela** opened a feature request to [discuss possible support of progressive web applications](https://github.com/sailfishos/sailfish-browser/issues/1000) (PWA). Notice that PWA are only fully supported by Chromium at the moment with a partial support for Firefox on Android but not on desktop.
+* `sailfish-browser`, the browser application itself, **rainemak** commented on **dcaliste**'s pull request to [introduce a long-tap in the tab list view](https://github.com/sailfishos/sailfish-browser/pull/959) for house-keeping actionss such as tab deletion. **mikaela** opened a feature request to [discuss possible support for progressive web applications](https://github.com/sailfishos/sailfish-browser/issues/1000) (PWAs). Notice that PWAs are only fully supported by Chromium at the moment with partial support for Firefox on Android but not on desktop.
```

Comment:
There actionss here written with two 's'.

Reply:
Thank you for noticing. Now fixed!